opennlp-models-langdetect/pom.xml

```diff
@@ -38,6 +38,7 @@
     <model.name>langdetect-183.bin</model.name>
     <model.version>1.8.3</model.version>
     <model.sha256>2ddf585fac2e02a9dcfb9a4a9cc9417562eaac351be2efb506a2eaa87f19e9d4</model.sha256>
+    <model.language>root</model.language>
```

Comment:
Just curious if there is any special meaning to `root`?

Reply:
Quoting from https://docs.oracle.com/en%2Fjava%2Fjavase%2F17%2Fdocs%2Fapi%2F%2F/java.base/java/util/Locale.html#ROOT

> The root locale is the locale whose language, country, and variant are empty ("") strings. This is regarded as the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations.

Reply:
TIL!